### PR TITLE
665 task endpoint tests

### PR DESCRIPTION
**Pull request by Mefpef**
Please DO NOT DELETE this template. Fill it. PR without your declaration that you licence your code under MIT will 
be discarded. 

Tips
----------------

 * Give the same name to the pull request as the name of ticket if possible
 * Make the title in form of: Pull request title (#<issue-id>)
 * Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 * Test your code throughly before PR.

Pull Request title
-------------------

     Write test for tasks API #665 (#665)

Pull Request description
-------------------------

    added test for task_endpoint

----

We need your declaration that your contribution is MIT licensed without it, we can not merge your code.
To make clear that you license your contribution under the MIT you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licensed under the [MIT license](https://opensource.org/licenses/MIT)

### CODE REVIEW

**Comment by DawidekZagajnik** [comment]
I'm taking look at this, once more: something isn't okay with branches - see how many commits you are pushing. Check on it.

File: test/api/endpoints/test_task_endpoint.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    if task is None:
+        task = {
+            "id": "test-id",
+            "name": "test-name",
+            "task_id": "test-task",
+            "timestamp": "2022-08-25T20:18:09.278Z",
+            "status": 'pending',
+            "progress": 1.1,
+            "type": "test",
+            "params": {}
```
Comment (DawidekZagajnik) [inline test/api/endpoints/test_task_endpoint.py:16]: you could fill in some params

Comment (DawidekZagajnik) [reply test/api/endpoints/test_task_endpoint.py:16]: to see if they persist

File: test/api/endpoints/test_task_endpoint.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+def test_should_add_new_task():
+    try:
+        response = _add_task()
+        result = response.json()
+
+        assert not result["errors"]
+        assert result["saved"] == 1
+        assert result["ids"] == ["test-id"]
+
+        task = {
```
Comment (DawidekZagajnik) [inline test/api/endpoints/test_task_endpoint.py:71]: seems to be responsibility of _add_task. This parameter should be passed there to be consistent

**Review by DawidekZagajnik — CHANGES_REQUESTED** [review]
Pretty good, though you could unify all tests into one function (adding task, checking loading, grouping, then deleting). If you did so, you could use one task: dict variable and instead of 
assert response["ids"] == ["test-id"], you could check for task["id"]. That would make maintenance and modifications easier.

**Review by DawidekZagajnik — APPROVED** [review]
seems ok

**Review by atompie — APPROVED** [review]

**Pull request merged**